warcraft/contracts/README.md

```diff
@@ -0,0 +1,62 @@
+This is a boilerplate `algob` project with ASA deployment and funding a smart contract (in TEAL & PyTEAL)
+
+To run the `sample-project`:
```

Comment:
Please update the readme to add info about the contracts.

Reply:
ok